src/Messages/OWSMessageSender.m

```diff
@@ -1224,27 +1189,6 @@ - (void)handleStaleDevicesWithResponse:(NSData *)responseData
     });
 }
 
-- (void)processException:(NSException *)exception
-         outgoingMessage:(TSOutgoingMessage *)message
-                inThread:(TSThread *)thread
-{
-    DDLogWarn(@"%@ Got exception: %@", self.tag, exception);
-
-    [self.dbConnection readWriteWithBlock:^(YapDatabaseReadWriteTransaction *transaction) {
```

Comment:
I believe this method was redundant. It did two things:

* Save a new error message, which was never initialized, so this had no effect.
* Mark a message send as failed.  We want to do this in `OWSSendMessageOperation`'s failure handler, not here.